solidity/test/TestPhasedEscrow.js

```diff
@@ -426,7 +425,378 @@ describe("PhasedEscrow", () => {
   }
 })
 
-describe("CurveRewardsEscrowBeneficiary", () => {
+describe("BatchedPhasedEscrow", () => {
```

Comment:
Could be worth extracting to a separate file.

Reply:
Yeah, I was struggling with it for a while. Though this is still a *phased* escrow, all beneficiaries, and all relationships (such as `PhasedEscrow` -> `BatchedPhasedEscrow`) are tested in this file. I'd prefer to keep it all together for now. WDYT?